04_setup_ironic.sh

```diff
@@ -204,6 +204,7 @@ if [ ! -f "${CACHED_MACHINE_OS_BOOTSTRAP_IMAGE}" ]; then
 fi
 
 # cached images to the bootstrap VM
+sudo -E podman pull --authfile $PULL_SECRET_FILE ${IRONIC_IMAGE} || echo "WARNING: Could not pull latest $IRONIC_IMAGE; will try to use cached images instead"
```

Comment:
nit: use consistent variable syntax (either $VAR or ${VAR}). Ideally also with "" around pull secret (may have spaces).

Reply:
Of course, just fixed it